tests/functional/object/mcg/test_cors_config_elements.py

```diff
@@ -721,7 +720,7 @@ def test_allowed_header_cors_element(
         )
         sample.wait_for_func_value(self.POSITIVE_RESPONSE)
 
-    @tier1
+    @tier2
```

Comment:
`tier` is missing for below tests
`test_basic_cors_operations`
`test_default_cors_pre_upgrade`